src/MICore/CommandFactories/MICommandFactory.cs

```diff
@@ -251,6 +251,13 @@ public virtual void DefineCurrentThread(int threadId)
             await _debugger.CmdAsync(command, ResultClass.running);
         }
 
+        public abstract Task Signal(string sig);
```

Comment:
Can we add an XML doc comment indicating what this does and the fact that it is optional?